exercises/rational-numbers/canonical-data.json

```diff
@@ -0,0 +1,302 @@
+{
+  "exercise": "rational-numbers",
+  "version": "1.0.0",
+  "comments": [
+    " The canonical data assumes mathematically correct real   ",
+    " numbers. The testsuites should consider rounding errors  ",
+    " instead of testing for exact values for any non-integer  ",
+    " tests.                                                   ",
+    " Rational numbers r are represented as arrays [a, b] so   ",
+    " that r = a/b.                                            "
+  ],
+  "cases": [
+    {
+      "description": "Arithmetic",
+      "cases": [
+        {
+          "description": "Addition",
+          "cases": [
+            {
+              "description": "Add two positive rational numbers",
+              "property": "add",
+              "r1": [1, 2],
+              "r2": [2, 3],
+              "expected": [7, 6]
+            },
+            {
```

Comment:
This is all looking pretty good I think.  Please take a look at #996.  It would be nice if you could make the necessary changes to conform to that policy  for example from "Add two positive rational numbers":
```JSON
"r1": [1, 2],
"r2": [2, 3],
```
would become:
```JSON
"input": {
  "r1": [1, 2],
  "r2": [2, 3]
},
```